.pre-commit-config.yaml

```diff
@@ -15,9 +15,11 @@ repos:
     rev: 6.0.0
     hooks:
       - id: flake8
-        args: ["--ignore=E501,W503"] # line too long and line before binary operator (black is ok with these)
+        args: ["--ignore=E501,W503,E231"] # line too long and line before binary operator (black is ok with these) and explicitly ignore the whitespace after colon error
         types:
           - python
+        # Suppress SyntaxWarning about invalid escape sequence from calitp-data-infra dependency without modifying source
+        entry: env PYTHONWARNINGS="ignore::SyntaxWarning" flake8
```

Comment:
Added `entry: env PYTHONWARNINGS="ignore::SyntaxWarning" flake8` to suppress a SyntaxWarning about invalid escape sequence from the calitp-data-infra dependency without modifying its source code